### PR TITLE
Updated Buidler to Hardhat (rebranded)

### PR DESCRIPTION
**Pull request by wackerow**
## Description
While updating git information for projects for another PR I came across Buidler which has been rebranded to Hardhat (only project listed that I found this type of change on)

This corrects all instances of:
- `buidler.dev` to `hardhat.org` (new link)
- `buidler` to `hardhat` (corrects the git repo links)
- `Buidler` to `Hardhat` (remainder of proper usage)
- renames `buidler.png` asset to `hardhat.png` (image theming seems to remain intact with rebranding), and updated query

#### Related Issue (none filed)
https://buidler.dev now redirects to this ->
image

### CODE REVIEW

File: src/content/developers/docs/development-networks/index.md [hunk 1]
```diff
@@ -39,14 +39,14 @@ Ganache provides both a desktop application (Ganache UI), as well as a command-l
 - [GitHub](https://github.com/trufflesuite/ganache)
 - [Documentation](https://www.trufflesuite.com/docs/ganache/overview)
 
-### Buidler EVM
+### Hardhat EVM
 
 A local Ethereum network designed for development. It allows you to deploy your contracts, run your tests and debug your code
 
-Buidler EVM come built-in with Buidler, a task runner for Ethereum smart contract developers.
+Hardhat EVM come built-in with Hardhat, a task runner for Ethereum smart contract developers.
```
Comment (samajammin) [inline src/content/developers/docs/development-networks/index.md:46]:
```suggestion
Hardhat EVM come built-in with Hardhat, an Ethereum development environment for professionals.
```

File: src/pages-conditional/developers/local-environment.js [hunk 2]
```diff
@@ -273,15 +273,15 @@ const frameworksList = [
...
-    id: "buidler",
-    url: "https://buidler.dev/",
+    id: "hardhat",
+    url: "https://hardhat.org/",
     background: "#2A2C32",
-    name: "Buidler",
+    name: "Hardhat",
     description:
-      "Buidler is a task runner for Ethereum smart contract developers.",
+      "Hardhat is a task runner for Ethereum smart contract developers.",
```
Comment (samajammin) [inline src/pages-conditional/developers/local-environment.js:284]:
```suggestion
      "Hardhat is an Ethereum development environment for professionals.",
```

**Review by samajammin — DISMISSED** [review]
LGTM! @fzeoli or @alcuadrado - mind giving this a review?

File: src/content/developers/docs/development-networks/index.md [hunk 3]
```diff
@@ -39,14 +39,14 @@ Ganache provides both a desktop application (Ganache UI), as well as a command-l
 - [GitHub](https://github.com/trufflesuite/ganache)
 - [Documentation](https://www.trufflesuite.com/docs/ganache/overview)
 
-### Buidler EVM
+### Hardhat Network
 
 A local Ethereum network designed for development. It allows you to deploy your contracts, run your tests and debug your code
 
-Buidler EVM come built-in with Buidler, a task runner for Ethereum smart contract developers.
+Hardhat EVM come built-in with Hardhat, an Ethereum development environment for professionals.
```
Comment (fzeoli) [inline src/content/developers/docs/development-networks/index.md:46]:
```suggestion
Hardhat Network comes built-in with Hardhat, an Ethereum development environment for professionals.
```

**Comment by fzeoli** [comment]
Left a single suggestion, and all the translations of the description need to be updated to the new tagline as well. Other than that, looks good!

**Comment by samajammin** [comment]
Thanks @fzeoli! I've committed your suggestion & will merge this.

Re: translations - I agree they should be updated but that's not something we can solve in this PR. we'll have to wait for the next language iterations to update those descriptions. Translations efforts are managed in Crowdin if you'd like to keep tabs:
https://crowdin.com/project/ethereumfoundation

**Review by samajammin — APPROVED** [review]

**Comment by samajammin** [comment]
@all-contributors please add @fzeoli for review.

**Pull request merged**